src/zh/toptoon/src/eu/kanade/tachiyomi/extension/zh/toptoon/Toptoon.kt

```diff
@@ -0,0 +1,193 @@
+package eu.kanade.tachiyomi.extension.zh.toptoon
+
+import eu.kanade.tachiyomi.network.GET
+import eu.kanade.tachiyomi.network.asObservableSuccess
+import eu.kanade.tachiyomi.source.model.FilterList
+import eu.kanade.tachiyomi.source.model.MangasPage
+import eu.kanade.tachiyomi.source.model.Page
+import eu.kanade.tachiyomi.source.model.SChapter
+import eu.kanade.tachiyomi.source.model.SManga
+import eu.kanade.tachiyomi.source.online.ParsedHttpSource
+import kotlinx.serialization.json.Json
+import kotlinx.serialization.json.decodeFromStream
+import okhttp3.Response
+import org.jsoup.nodes.Document
+import org.jsoup.nodes.Element
+import rx.Observable
+import uy.kohesive.injekt.injectLazy
+import java.text.ParseException
+import java.text.SimpleDateFormat
+import java.util.Locale
+
+class Toptoon : ParsedHttpSource() {
+    override val name: String = "Toptoon頂通"
```

Comment:
```suggestion
    override val name: String = "TOPTOON頂通"
```

Website seems to use all caps.